app/src/nxt/java/com/pileproject/drive/execution/NxtController.java

```diff
@@ -208,17 +211,18 @@ public void setMotorPower(MotorKind kind, double percent) {
 
         switch (kind) {
             case LeftMotor:
-                mLeftMotorPower = (int) ((MAX_MOTOR_POWER * percent) / 100.0);
+                mLeftMotorPower = (int) percent;
```

Comment:
`percent` is a misleading name. They are used as `speed` actually. I will fix the type/name of them. Thanks :)